docs/section4/4-5.md

```diff
@@ -0,0 +1,42 @@
+# Airside Systems – Rule 4-5
+
+**Rule ID:** 4-5
+ 
+**Rule Description:** HVAC fans shall remain on during unoccupied hours in spaces that have health and safety mandated minimum ventilation requirements during unoccupied hours.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #2 for the proposed building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. Applies to all HVAC systems that serve zones that have health and safety mandated minimum ventilation requirements during unoccupied hours. 
+ 
+**Function Call:** 
+
+1. get_zones_health_safety_vent_reqs()
+
+**Manual Check:** None  
+  
+**Rule Logic:**
+**Applicability Check 1:** 
+- Get list of zones that are likely to have health and safety mandated minimum ventilation requirements during unoccupied hours: `applicable_zones_health_safety_unocc_list_p = get_zones_health_safety_vent_reqs(P_RMR)`
+- For each zone in P_RMR: `for zone_p in P_RMR..Zone:`
+    - Reset health_safety_unocc_check boolean variable which indicates if a zone likely has health and safety reqs during unocc periods: `health_safety_unocc_check = FALSE`
+    - Check if the zone is likely to have health and safety reqs during unocc by checking if the zone is in the list of applicable zones, if yes then set health_safety_unocc_check boolean variable to true: `if zone_p.id in applicable_zones_health_safety_unocc_list_p: health_safety_unocc_check = TRUE`
+    - Check if health_safety_unocc_check boolean variable equals true then carry on with rule logic if true: `if health_safety_unocc_check == TRUE:`
+        - For each terminal in zone_p: `for terminal_p in zone_p.terminals:`
+            - Get HVAC system serving terminal: `hvac_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_p as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_p = heating_ventilation_air_conditioning_systems_list_p.append(heating_ventilation_air_conditioning_systems_p)`                    
+        - Add to master list of HVAC system that are applicable to this zone after looping through the terminal units: `applicable_hvac_systems_list_p = applicable_hvac_systems_list_p.append(heating_ventilation_air_conditioning_systems_list_p)`         
+- Convert the list of applicable_hvac_systems_list_p to a set and the back to a list to eliminate duplicates after looping through all zones: `applicable_hvac_systems_list_p = list(set(applicable_hvac_systems_list_p))`                             
+- For each hvac_p in applicable_hvac_systems_list_p: `for each hvac_p in applicable_hvac_systems_list_p:`                         
```

Comment:
1. I wonder if we can replace the above by just saying to locate all HeatingVentilationAirConditioningSystems that serve at least one zone returned by get_zones_health_safety_vent_reqs(P_RMR).
2. I believe the rule for continuous operation only applies to HVAC systems that serve applicable zones AND provide ventilation, so I think we also need to add a check for the corresponding FanSystem.minimum_outdoor_airflow>0

Reply:
good point about checking for only systems that provide ventilation.

Reply:
as discussed on 2022.03.16 RDS call, will create a function to get HVAC systems based on zone list and use that here.